pkg/pillar/cmd/zedrouter/dnsmasq_test.go

```diff
@@ -5,13 +5,24 @@ package zedrouter
 
 import (
 	"bytes"
+	"fmt"
+	"math"
 	"net"
+	"os"
+	"os/exec"
+	"path/filepath"
 	"regexp"
+	"sync"
+	"sync/atomic"
 	"testing"
+	"time"
 
+	"github.com/digineo/go-dhclient"
```

Comment:
Seems like this import doesn't match the use based on the golangcilint errors. I assume the build (make test) also failed due to this but I haven't checked.

Reply:
It is worse - I did not check the license beforehand and it is GPLv3 ...

Reply:
Is there an alternative which has a better license? Even though it is only used by the test code, I suspect the SBOM will include it as being part of the built image (you can check with make sbom and as @deitch if make sbom has issues.)

Reply:
I switched to "github.com/insomniacslk/dhcp/dhcpv4/nclient4" which we're also using in https://github.com/lf-edge/rol/blob/main/src/go.mod#L12